ballerina/build.gradle

```diff
@@ -311,11 +317,14 @@ task packageDistZip(type: Zip) {
     /* Standard Libraries */
     def birCache = new File("${parentDir}/distributions/jballerina-${ballerinaVersion}/cache/bir/")
     def baloCache = new File("${parentDir}/distributions/jballerina-${ballerinaVersion}/cache/balo/")
+    def jarCache = new File("${parentDir}/distributions/jballerina-${ballerinaVersion}/cache/jar/")
```

Comment:
So are we copying stdlib jar to cache/jar directory and bre/lib both? Then there will be duplicate stdlib thin jars in the distribution.

Reply:
Yes, currently we have to duplicate because we can't break the current implementation. After we totally moved to the new cache, we can remove that duplication.

Reply:
Till we do the packerina side changes we need to pack at both places